radicle-cli/src/commands/auth.rs

```diff
@@ -65,7 +66,10 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
 }
 
 pub fn init(options: Options) -> anyhow::Result<()> {
-    term::headline("Initializing your 🌱 profile and identity");
+    term::headline(format!(
+        "Initializing your {} 🌱 identity",
+        term::format::highlight("radicle")
```

Comment:
Style question: how come we don't capitalise "radicle"?

I think it's actually a mixed bag for if we do or don't. Just noticed a "Radicle" below. I've been thinking it would be a good idea to have a sort of style guide for the CLI output to be more consistent when editing/adding commands.

Reply:
Yeah, good question. I've been waffling about this one. I think it looks better lowercase usually, in the terminal context. A style guide is a good idea, though we may be able to encode all of it in functions/consts, eg. for this we could have `terminal::format::RADICLE` or something, and it would be the correct color/capitalization. Same for RIDs, NIDs etc.

Reply:
Ya, that'd be a great idea!